pages/about-us/newsletter/2013/07/en.md

```diff
@@ -28,18 +28,16 @@ For anyone else who supports these views, please stop using Riseup.
 
 And to read some smarter and more in-depth thoughts on gender, transphobia, and being transgendered, here are some great links (in English):
 
-"on the socialization of children":http://www.questioningtransphobia.com/?p=2884
+* on the socialization of children http://www.questioningtransphobia.com/?p=2884
```

Comment:
these should be links, this seems a common translation problem, it got turned into:

on the socialization of children http://www.questioningtransphobia.com/?p=2884

instead of making 'on the socialization of children' being a link to http://www.questioningtransphobia.com/?p=2884

Reply:
Newsletters are in plaintext, which leads that most of the links in them are just simple calls as this one. When I had to figure it out what should I do with them I just followed the rest of the pages which had no links.